src/seedu/addressbook/data/person/NameComparator.java

```diff
@@ -0,0 +1,14 @@
+package seedu.addressbook.data.person;
+
+import java.util.Comparator;
+
+public class NameComparator implements Comparator<Person> {
```

Comment:
Missing header comment. All non trivial classes should have javadoc format header comments.